package.json

```diff
@@ -23,38 +23,36 @@
     "url": "https://github.com/cerner/generator-terra-module/issues"
   },
   "dependencies": {
-    "chalk": "^1.0.0",
+    "chalk": "^2.4.1",
     "lodash.camelcase": "^4.3.0",
     "lodash.startcase": "^4.4.0",
     "lodash.upperfirst": "^4.3.1",
-    "yeoman-generator": "^0.22.0",
-    "yosay": "^1.0.2"
+    "yeoman-generator": "^3.1.1",
+    "yosay": "^2.0.2"
   },
   "devDependencies": {
-    "eslint": "^2.1.0",
-    "eslint-config-xo-space": "^0.10.0",
-    "gulp": "^3.9.0",
-    "gulp-eslint": "^2.0.0",
-    "gulp-exclude-gitignore": "^1.0.0",
-    "gulp-istanbul": "^0.10.3",
-    "gulp-line-ending-corrector": "^1.0.1",
-    "gulp-mocha": "^2.0.0",
-    "gulp-nsp": "^2.1.0",
-    "gulp-plumber": "^1.0.0",
-    "slugg": "^1.0.0",
-    "yeoman-assert": "^2.0.0",
+    "eslint": "^5.0.0",
+    "eslint-config-terra": "^2.0.0",
+    "mocha": "^5.2.0",
+    "yeoman-assert": "^3.0.0",
     "yeoman-test": "^1.0.0"
```

Comment:
any reason why you didn't choose to update `yeoman-test`?

Reply:
Ah, I missed that 2.0.0 was available, I'll get that updated in this PR.

Reply:
Updated: https://github.com/cerner/generator-terra-module/pull/110/commits/fd9e1b154f01ea899452d526013866202efe4064